### ISSUE TITLE
`*.vyne.co` links broken

### ISSUE THREAD

**Issue by tripplilley** [taxilang/taxilang#4]
Various links in the docs to sites under `vyne.co` are broken (presumably because you're no longer under Vyne?).

Specifically, https://voyager.vyne.co/ and https://blog.vyne.co/ links no longer resolve.

**Comment by martypitt** [comment]
Thanks for reporting this.

Yeah, we've renamed from Vyne to Orbital.
I've updated the links, but some issues remain, primarily because

 * Voyager isn't live yet (will try hard to get this back up this week at voyager.orbitalhq.com)
 * We haven't migrated the blogs from the old Vyne blog to the Orbital blog.  I'll bring the Voyager one across as part of the relaunch of Voyager, and others on an as-needed basis.
